ocaml/xapi/xapi_xenops.ml

```diff
@@ -653,7 +653,7 @@ module MD = struct
     {
       domain = 0000;
       bus = 0;
-      dev = int_of_string device;
```

Comment:
should we move `11` to the xapi_globals, and give it a meaningful name, as it looks important.

Reply:
Yes, good idea.

Reply:
I've updated it.

Reply:
sorry, I cannot find it, did you forgot to push your commit?

Reply:
I had pushed the wrong one :) It's there now...